iceoryx2-ffi/ffi/src/config.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2024 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Apache Software License 2.0 which is available at
+// https://www.apache.org/licenses/LICENSE-2.0, or the MIT license
+// which is available at https://opensource.org/licenses/MIT.
+//
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+#![allow(non_camel_case_types)]
+
+use iceoryx2::prelude::*;
+
+// BEGIN type definition
+
+pub type iox2_config_t = *const Config;
+
+// END type definition
+
+// BEGIN C API
+#[no_mangle]
+pub extern "C" fn iox2_config_get_global() -> iox2_config_t {
```

Comment:
Style question: For getter methods I followed the style without `get`. This function has the `get` though, similar to the `Rust` function. Shall we keep this? Shall I rename this to `iox2_config_get_global_config`?

Reply:
My proposal would be to rename the rust function to `Config::global_config()`. This is code from the early steps of iceoryx2 where I followed the `get_` and `set_` prefix style that I applied in C++.

Reply:
Will do this in a follow up